apps/web/src/server/applications/case/documentation/applications-documentation.controller.js

```diff
@@ -308,25 +334,27 @@ export async function removeApplicationsCaseDocumentationPublishingQueue(request
 /**
  * Handle unpublishing a document
  *
- * @type {import('@pins/express').RenderHandler<{}, {}, {}, {}, {caseId: string, documentGuid: string}>}
+ * @type {import('@pins/express').RenderHandler<{}, {}, {documentGuids: string[]}, {}, {}>}
  * */
-export async function postUnpublishDocument({ params }, response) {
-	const { caseId, documentGuid } = params;
+export async function postUnpublishDocuments({ body }, response) {
+	const { documentGuids } = body;
+	const { caseId } = response.locals;
+
+	const { errors } = await unpublishCaseDocumentationFiles(caseId, documentGuids);
 
-	const { errors } = await unpublishCaseDocumentationFiles(Number(caseId), [documentGuid]);
```

Comment:
handle multiple unpublishing.

also, using the caseId from response.locals prevents the need for casting into a Number